gagbo-truly-cross-platform.md

```diff
@@ -0,0 +1,137 @@
+Make your setup truly cross-platform
+===============================================================================
+Once you have taken the time to setup Vim exactly the way you want, you might
+still encounter configuration issues. My whole `~/.vim` folder is under version
+control so I can just clone it on any computer I want my familiar Vim
+experience.
+
+But I use Windows computers and Linux computers. Even within the
+same OS, I might have different dependencies installed for the plugins I try to
+import. Also, from time to time I like to use Neovim too, to try out a few
+unique features.
+
+
+
+OS specific settings
+-------------------------------------------------------------------------------
```

Comment:
I think every mention of "OS" should be "environment".

When using the Vim provided by Git Bash on Windows, your OS is still Windows but the "environment" in which Vim runs is likely to be MingW. And you don't make the same decisions for a "Windows" environment as you do for a "MingW" environment.

Hence `g:env` versus `g:os` (which was its initial name in my config BTW).

Reply:
Agreed I'll change this. I wanted to separate your gist in multiple steps since the caching into `g:env` is a little "off-topic" when presenting the function